Add Known Null PowerShell issue for ReadMe (#625)
Azure/azure-functions-sql-extension: README.md

    * Use [dynamic (imperative)](https://learn.microsoft.com/azure/azure-functions/functions-bindings-expressions-patterns#binding-at-runtime) bindings (.NET only)
    * Use [IAsyncCollector](https://learn.microsoft.com/azure/azure-functions/functions-dotnet-class-library?tabs=v2%2Ccmd#writing-multiple-output-values) and call `FlushAsync` in the order desired (.NET only)
- For PowerShell Functions that use hashtables must use the `[ordered]@` for the request query or request body assertion in order to upsert the data to the SQL table properly. An example can be found [here](https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/samples-powershell/AddProductsWithIdentityColumnArray/run.ps1).

> - PowerShell Functions using Output bindings can not pass in null or empty values via the query string. Workaround is to use the `$TriggerMetadata[$keyName]` to retrieve the query property. Issue is tracked [here](https://github.com/Azure/azure-functions-powershell-worker/issues/895).

Reviewer: Would suggest linking to https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/samples-powershell/AddProductParams/run.ps1 as an example of this workaround.

> - PowerShell Functions using Output bindings can not pass in null or empty values via the query string. Workaround is to use the `$TriggerMetadata[$keyName]` to retrieve the query property. Issue is tracked [here](https://github.com/Azure/azure-functions-powershell-worker/issues/895).

Reviewer 2: ```suggestion
- PowerShell Functions using Output bindings cannot pass in null or empty values via the query string. Workaround is to use the `$TriggerMetadata[$keyName]` to retrieve the query property. Issue is tracked [here](https://github.com/Azure/azure-functions-powershell-worker/issues/895).
```
